pkg/kubelet/dockershim/docker_container.go

```diff
@@ -254,6 +254,10 @@ func (ds *dockerService) RemoveContainer(containerID string) error {
 	// we can't prevent that now, so we also clean up the symlink here.
 	err := ds.removeContainerLogSymlink(containerID)
 	if err != nil {
+		// no-op if container has already been removed.
+		if containerNotExistErr(err) {
```

Comment:
yep, It is processed inside, if inspect container failed or remove log symlink failed, will return error. But it returned all the error message, so it make sense to match error message here. While I don't think it's pretty, do you have any good advice?